components/Banner/src/MDCBannerView.h

```diff
@@ -51,4 +51,9 @@ __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
  */
 @property(nonatomic, readonly, strong, nonnull) MDCButton *trailingButton;
 
+/**
+ A Boolean value that controls whether the divider below the banner is visible.
+ */
+@property(nonatomic, readwrite, assign) BOOL showsDivider;
```

Comment:
I think we may find this to be more flexible if we follow CALayer’s lead and expose both the height and color properties of the border. We then would not need this Boolean because the height will default to zero and the color will default to “some reasonable default”.

What do you think?

Reply:
I think that's a good idea and makes the color and height configurable. I can change this API to two APIs (including color, height). For internal implementation, because it is fully using Autolayout, adding CALayer for now would require adding some frame update in view life cycle methods. I can still use UIView internally though it doesn't get the default zero height one. We can always change the implementation. How do you feel about it, especially API-wise?

Reply:
Ah using a UIView internally to represent the border still would be fine; you’ll just map the public color property to this border view’s background color, for example.

Reply:
I thought about it again. The height property is not configurable from design for now. We can expose the color. I am thinking if we can directly expose divider as a UIView following how the API of Banner is designed ([Header](https://github.com/material-components/material-components-ios/blob/develop/components/Banner/src/MDCBannerView.h) for reference)?

Reply:
I originally considered this property was more similar to `showsVerticalScrollIndicator`.

Reply:
As an alternative: `@property(nullable, nonatomic, strong) UIView *dividerView;` ?

Reply:
Let's just expose the color property then for now. If no color is provided, then no border will be shown.

Reply:
Spoke offline with @wenyuzhang666. We discussed that providing a `showsDivider` and a `dividerColor` property is an acceptable path forward to ensure a balance of clarity + configurability for our theming extensions.

Reply:
Addressed in fb8d2f45f074976253daa447d45fd55211b20b64.